scan-framework/src/main/java/com/getbouncer/scan/framework/image/NV21Image.kt

```diff
@@ -30,7 +30,7 @@ import kotlin.experimental.inv
 /**
  * Get the RenderScript instance.
  */
-val getRenderScript = memoize { context: Context -> RenderScript.create(context) }
```

Comment:
memoize holds onto a reference of the parameters used to call the function for later comparison. This should probably be changed so it holds onto the object hash or something (and handles collisions cleanly).